onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -981,6 +1002,12 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.input.shape) == 0,
         reason="fixme: Rank(0) input will lead ORT failed due to different rank(result) in if-else branch",
     ),
+    skip(
+        "scatter_reduce",
+        matcher=lambda sample: sample.kwargs.get("include_self") is False
+        or len(sample.input.shape) == 0,  # skip rank 0 case
```

Comment:
Does it fail when rank is 0?

Reply:
Yes. Added more information.
After convert the self/index/src to rank 1 tensor,  will still failed.